components/application-service/base/kustomization.yaml

```diff
@@ -13,3 +13,12 @@ namespace: application-service
 
 patches:
   - ./manager_resources_patch.yaml
+
+# APIExport managed by infra-deployments for now
+patchesStrategicMerge:
+- |-
+  apiVersion: apis.kcp.dev/v1alpha1
+  kind: APIExport
+  metadata:
+    name: has
+  $patch: delete
```

Comment:
Just asking, but why couldn't we delete the `has` APIExport from the application-service repo (for example as part of https://github.com/redhat-appstudio/application-service/pull/192)
I thought that our goal is to have one APIExport that would be used as the only source of truth. Or am I missing something?

Reply:
Removed the removing part from this PR, handled in #790  for has